recipes-bsp/jh7110-spl-tool/jh7110-spl-tool_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "spl_tool is a jh7110 signature tool used to generate spl header information and generate u-boot-spl.bin.normal.out."
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e6dc25dc2418b8831c906d43809d8336"
+SECTION = "BSP"
+
+SRCREV = "8c5acc4e5eb7e4ad012463b05a5e3dbbfed1c38d"
+SRC_URI = "git://github.com/starfive-tech/Tools;protocol=https;branch=master"
+
+inherit native
+
+S = "${WORKDIR}/git/spl_tool"
+
+do_compile () {
+    cd ${S}
+    oe_runmake clean
+    oe_runmake
+}
+
+do_install () {
+    cd ${S}
+    install -d ${D}${bindir}
+    install -m 0755 spl_tool ${D}${bindir}/spl_tool
+}
+
+BBCLASSEXTEND = "native"
```

Comment:
this seems redundant. You are already inheriting native on line 9

Reply:
That's correct. But without `BBCLASSEXTEND = "native"` here, `DEPENDS:append:visionfive2 = " jh7110-spl-tool-native"` in U-Boot didn't work. I will re-check again

Reply:
this is not addressed in v2 ?

Reply:
I've removed "inherit native" from line 9. Only BBCLASSEXTEND us kept.